lib/chutney/TorNet.py

```diff
@@ -1117,6 +1149,30 @@ def hup(self):
         print("Sending SIGHUP to nodes")
         return all([n.getController().hup() for n in self._nodes])
 
+    def wait_for_bootstrap(self):
+        print("Waiting for nodes to bootstrap...")
+        limit = 20 #bootstrap time
```

Comment:
This number should be configurable.
We could use CHUTNEY_START_TIME?
https://gitweb.torproject.org/chutney.git/tree/README#n130